test/e2e/common/container_probe.go

```diff
@@ -211,7 +211,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		Testname: Pod liveness probe, docker exec, restart
 		Description: A Pod is created with liveness probe with a Exec action on the Pod. If the liveness probe call  does not return within the timeout specified, liveness probe MUST restart the Pod.
 	*/
-	ginkgo.It("should be restarted with a docker exec liveness probe with timeout ", func() {
+	ginkgo.It("should be restarted with an exec liveness probe with timeout [NodeConformance]", func() {
```

Comment:
perhaps change the `Release` to 1.20 for this test as well?

![image](https://user-images.githubusercontent.com/9950081/99596381-041ba880-29ab-11eb-99fe-196b88a1040c.png)

Reply:
Yeah I'm not sure what the best practice here is. The test was added in v1.9, but it was skipped and never ran until v1.20.

Reply:
this is why I am recommending the change, but either way is ok I think. Also not aware on the best practices here.

/lgtm